components/director/internal/domain/tenant/resolver.go

```diff
@@ -278,7 +279,13 @@ func (r *Resolver) Write(ctx context.Context, inputTenants []*graphql.BusinessTe
 		return nil, err
 	}
 
-	r.syncSystemsForTenants(ctx, tenantIDs)
+	tenantIDs := make([]string, 0, len(tenantsMap))
+	for tenantID, tenantType := range tenantsMap {
+		tenantIDs = append(tenantIDs, tenantID)
+		if tenantType == tenantpkg.Account || tenantType == tenantpkg.Customer {
```

Comment:
Can you create a function that determines that? Something like: 
```
func isSyncableTenant(tenantType tenantpkg.Type) {
   return tenantType == tenantpkg.Account || tenantType == tenantpkg.Customer
}
```

then use it on L286 and L314